mlx-rs/src/ops/shapes.rs

```diff
@@ -677,13 +695,14 @@ impl<'a> PadWidth<'a> {
 /// use mlx_rs::{prelude::*, ops::*};
 ///
 /// let a = Array::from_iter(0..4, &[2, 2]);
-/// let result = pad(&a, 1, Array::from_int(0));
+/// let result = pad(&a, 1, Array::from_int(0), PadMode::Constant);
```

Comment:
Is this the only place where `PadMode` is used? If so, should we just make `PadMode::Constant` the default value?